src/NClient.CodeGeneration/NClient.CodeGeneration.Facades.NSwag/Templates/Facade.liquid

```diff
@@ -19,20 +27,27 @@ public partial interface I{{ FacadeName }}Client : I{{ FacadeName }}Facade
 {%         if operation.IsDeprecated -%}
     [System.Obsolete]
 {%         endif -%}
-{% template Controller.Method.Annotations %}
+{% template Facade.Method.Annotations %}
+{%         if GenerateFacades -%}
     [{{ NClientAnnotationsNamespace }}.Override]
     new {{ operation.ClientResultType }} {{ operation.ActualOperationName }}Async({% for parameter in operation.Parameters %}{{ parameter.TypeInControllerInterface }} {{ parameter.VariableName }}{% if GenerateOptionalParameters and parameter.IsOptional and parameter.HasDefault == false %} = null{% endif %}{% if parameter.IsLast == false or UseCancellationToken %}, {% endif %}{% endfor %}{% if UseCancellationToken %}System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken){% endif %});
+{%         else %}
+    [{{ NClientAnnotationsHttpNamespace }}.{{ operation.HttpMethodUpper }}Method("{{ operation.Path }}"{% if operation.HasRouteName %}, Name = "{{ operation.RouteName }}"{% endif %})]
+    {{ operation.ClientResultType }} {{ operation.ActualOperationName }}Async({% for parameter in operation.Parameters %}{% if parameter.IsQuery %}[{{ NClientAnnotationsHttpNamespace }}.QueryParam{% if parameter.IsValidIdentifier == false %}(Name = "{{ parameter.Name }}"){% endif %}] {% endif %}{% if parameter.IsHeader %}[{{ NClientAnnotationsHttpNamespace }}.HeaderParam{% if parameter.IsValidIdentifier == false %}(Name = "{{ parameter.Name }}"){% endif %}] {% endif %}{% if parameter.IsBody and parameter.IsBinaryBody == false %}[{{ NClientAnnotationsHttpNamespace }}.BodyParam] {% endif %}{% if GenerateModelValidationAttributes and parameter.IsRequired %}[{{ RequiredAttributeType }}] {% endif %}{{ parameter.TypeInControllerInterface }} {{ parameter.VariableName }}{% if GenerateOptionalParameters and parameter.IsOptional and parameter.HasDefault == false %} = null{% endif %}{% if parameter.IsLast == false or UseCancellationToken %}, {% endif %}{% endfor %}{% if UseCancellationToken %}System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken){% endif %});
+{%         endif -%}
 
 {%      endfor -%}
 }
+{% endif -%}
 
+{% if GenerateFacades -%}
```

Comment:
I think we should separate this on two diff templates.

Reply:
The client and the facade are linked by inheritance and logic. It is reasonable to put them in the same file as in the NSwag library